.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@Julio-Arroyo: I think this directory was pushed accidentally. Could you please remove it?

Reply:
Done